react-example/cypress/integration/commerce/commerce.spec.ts

```diff
@@ -32,7 +32,7 @@ describe('Track Purchase', () => {
         method: 'POST',
         url: '/api/commerce/trackPurchase*'
       },
-      { fixture: 'commerce/400.json' }
+      { fixture: 'commerce/400.json', statusCode: 400 }
```

Comment:
added these because previously the Cypress /inApp stub requests were 200ing when they should have been 400ing